cmd/minishift/cmd/util/oc.go

```diff
@@ -150,20 +150,19 @@ func GetOcPathForProfile(profileName string) (error, string) {
 // PullOpenshiftImageAndCopyOcBinary pull the openshift image if not available and then copy the client binary to Remote/VM machine
 // Any occurring error is also returned.
 func PullOpenshiftImageAndCopyOcBinary(dockerCommander docker.DockerCommander, requestedOpenShiftVersion string) error {
-	fmt.Printf("-- Pulling the Openshift Container Image ")
-	progressDots := progressdots.New()
-	progressDots.Start()
+
```

Comment:
This is something @gbraad specifically asked so I am not sure why we are removing it?

Reply:
@praveenkumar We are not removing it but moving it where it is required. If the image does not exist then the messages shows up in the console. E.g.

```
$ minishift start
-- Starting profile 'minishift'
-- Check if deprecated options are used ... OK
-- Checking if https://github.com is reachable ... OK
-- Checking if requested OpenShift version 'v3.9.0' is valid ... SKIP
-- Checking if requested OpenShift version 'v3.9.0' is supported ... OK
-- Checking if requested hypervisor 'xhyve' is supported on this platform ... OK
-- Checking if xhyve driver is installed ...
   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
   Checking for setuid bit ... OK
-- Checking the ISO URL ... OK
-- Checking if provided oc flags are supported ... OK
-- Starting the OpenShift cluster using 'xhyve' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB
-- Starting Minishift VM ............... OK
-- Checking for IP address ... OK
-- Checking for nameservers ... OK
-- Checking if external host is reachable from the Minishift VM ...
   Pinging 8.8.8.8 ... OK
-- Checking HTTP connectivity from the VM ...
   Retrieving http://minishift.io/index.html ... OK
-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 1% used OK
-- OpenShift cluster will be configured with ...
   Version: v3.9.0
-- Pulling the Openshift Container Image ................... OK
-- Copying oc binary from the OpenShift container image to VM ... OK
```

Reply:
If the image exists, it does not showup

```
$ minishift start

-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 1% used OK
   Importing 'openshift/origin:v3.9.0' .......... OK
   Importing 'openshift/origin-docker-registry:v3.9.0' .. OK
   Importing 'openshift/origin-haproxy-router:v3.9.0' .. OK
-- OpenShift cluster will be configured with ...
   Version: v3.9.0
-- Copying oc binary from the OpenShift container image to VM ... OK
```

Reply:
@LalatenduMohanty As of now following is the behavior in case the image exported and exist. Why do you think that's not correct one?

```
[...]
-- Checking available disk space ... 1% used OK
   Importing 'openshift/origin:v3.9.0' .............. OK
   Importing 'openshift/origin-docker-registry:v3.9.0' ... OK
   Importing 'openshift/origin-haproxy-router:v3.9.0' .... OK
-- OpenShift cluster will be configured with ...
   Version: v3.9.0
-- Pulling the Openshift Container Image . EXISTS
-- Copying oc binary from the OpenShift container image to VM .... OK
[...]
```

Reply:
@praveenkumar Because just before that we are importing images.  So it was confusing for me to see that after importing images whey we are pulling docker image. The Idea behind using cached images was that we do not need to pull images from internet and it was kind of confusing. Also personally I think we only need the message when image is not available.

Reply:
> Because just before that we are importing images. So it was confusing for me to see that after importing images whey we are pulling docker images.

It just the message and which showing the correct result not sure why it confuses you? I just added @gbraad for review who initially suggested me to do so.

Reply:
When you do not have any cached images, it will show the pull...

The `EXISTS` is to indicate an easy debug state; the image existed and network was not used to pull